src/gui/PasswordEdit.cpp

```diff
@@ -105,3 +114,41 @@ void PasswordEdit::autocompletePassword(const QString& password)
         setText(password);
     }
 }
+
+/**
+ * Set the status of secure input entry on macOS. This stops keyboard intercept
+ * processes (e.g. keyloggers, accessibility services) from reading keypresses.
+ *
+ * It's important to turn off this off when not needed to avoid interfering with
+ * accessibility functionality and other legitimate uses of keyboard event taps.
+ *
+ * See the Apple Technical Note 2150:
+ * https://developer.apple.com/library/archive/technotes/tn2150/_index.html
+ *
+ * @param b Status to set secure input entry to
```

Comment:
Addition: you said you copied from Qt4? If so, please mark the source.

Reply:
Done.